modules/exploits/linux/http/geutebruck_testaction_exec.rb

```diff
@@ -0,0 +1,84 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Geutebruck testaction.cgi Remote Command Execution',
+      'Description'    => %q{
+        This module exploits a an arbitrary command execution vulnerability. The
+        vulnerability exists in the /uapi-cgi/testaction.cgi page and allows an
+        authenticated user to execute arbitrary commands with root privileges.
+        with web user privileges. Firmware <= 1.12.14.5 are concerned.
```

Comment:
`with root privileges` and `with web user privileges` appear to be conflicting statements.

Reply:
oops ;) root privs !

Reply:
Resolved with recent commits.